common/locales/en/settings.json

```diff
@@ -140,5 +140,8 @@
     "gemCapExtra": "Gem Cap Extra:",
     "mysticHourglasses": "Mystic Hourglasses:",
     "paypal": "PayPal",
-    "amazonPayments": "Amazon Payments"
+    "amazonPayments": "Amazon Payments",
+    "timezone": "Timezone",
+    "timezoneUTC": "Your timezone is: <b><%= utc %></b>",
```

Comment:
I'd like it to say something more like "Habitica has detected from your PC / mobile device that your timezone is ..." or "Habitica has been informed by your PC / mobile device that your timezone is ...".

That makes it clearer where the problem lies when the value is wrong, and prevent Habitica appearing to be incorrect.